src/helperFunctions/getDateInString.ts

```diff
@@ -1,7 +1,7 @@
 const getDateInString = (date: Date) => {
     const year = date.getFullYear()
-    const month = date.getMonth() + 1;
-    const day = date.getDate();
+    const month = ("0" + (date.getMonth() + 1)).slice(-2);
```

Comment:
Would ternary operator be better suitable here?
Checking the length and adding 0 or not accordingly?
A new dev find it difficult to understand without comments or context

Reply:
Like this?
const a = new Date();
const month = a.getMonth > 9 ? a.getMonth : ("0" + (a.getMonth() + 1)).slice(-2);

Reply:
@DeepanshuSarswat Please try to use code blocks in the comments. It helps with readability.

@harshith-venkatesh I actually really like the solution used here. A comment would help. I feel a ternary would be an overkill.

```js
const month = ("0" + (date.getMonth() + 1)).slice(-2);
```

Reply:
True, after taking a look at the comment back, I felt the same as well.
However the comment do play an important role here

Reply:
@harshith-venkatesh I feel at the end of the day, we are not writing code for new devs. We are writing code that is easy to read, proven correct (Most important) and reason about.

A new dev should take the time to look up what the language functions do and if they can guarantee that the code will always produce the correct results.

If we always cater to the new dev, we will be stuck writing verbose basic code.

Reply:
Can't disagree with your comment. That intention here was, I know the reason behind this fix.
But a person new to review or with no context should also be able to easily review and approve right.
I can understand, this statement could be easily deciphered, was just presenting my thoughts here🙂👍